app/components/user-search.cjsx

```diff
@@ -14,7 +14,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: "#{value}*", page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
+            { value: user.id, label: "@#{ user.login }" }
```

Comment:
The display_name won't have changed from what it was previously so it'll probably be the easiest way to find people for now.